### ISSUE TITLE
This plugin no longer works on cordova-android >=7. Fixable in one line, pls do it.

### ISSUE THREAD

**Issue by kakysha** [EddyVerbruggen/Custom-URL-scheme#294]
This guy figured out why this plugin stop working on cordova-android>=7 https://github.com/bytelabsco/Custom-URL-scheme, can you @EddyVerbruggen pls merge his changes? So many open issues with such an easy fix.

**Comment by EddyVerbruggen** [comment]
Happy to, but I'd like to know if that will break compatibility with older Cordova versions. Any idea? Or can you/someone test it?

**Comment by kakysha** [comment]
It probably does, cause they moved AndroidManifest.xml in recent versions, so older versions of cordova should still have it at previous location.

**Comment by EddyVerbruggen** [comment]
Yeah, that's what doesn't really make it an easy fix. We could create a new major release and add a note to the readme..

**Comment by panoti** [comment]
I forked a temporary use while waiting for the new version.

**Comment by tobiaswaltl** [comment]
Has there been any progress on this issue?

**Comment by Phunky** [comment]
@kakysha thanks for this post, the fork is working for us on Android where as the current release on this repo is not.
